config/puma.rb

```diff
@@ -24,6 +24,11 @@
   end
 end
 
+# Write Puma logs to file in append mode
+if ENV['PUMA_LOG'] && ENV['PUMA_LOG_ERR']
+  stdout_redirect ENV['PUMA_LOG'], ENV['PUMA_LOG_ERR'], true
+end
```

Comment:
what if we just required one of them, default the other to `/dev/null`?
```suggestion
if ENV['PUMA_LOG']
  stdout_redirect ENV['PUMA_LOG'], (ENV['PUMA_LOG_ERR'] || '/dev/null'), true
end
```

Reply:
I think we'll want both or neither to start. It's early days, but I suspect we'd want to output to STDOUT/STDERR either way since containers will be picking up logs through that.

Reply:
ok! sounds good